### PR TITLE
Move out most of "non-fully active" handling guide

### PR DESCRIPTION
**Pull request by rakina**
The guide is being moved to a separate "Supporting BFCached documents" doc, to keep the design principles doc short. Replaces the previous content with a short intro and link to the full guide (which is still in my personal repo). See also https://github.com/w3ctag/design-reviews/issues/628#issuecomment-1227165134 for context.

***
<a href="https://pr-preview.s3.amazonaws.com/rakina/design-principles/pull/392.html" title="Last updated on Dec 6, 2022, 8:48 AM UTC (f314c87)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/392/a6266b8...rakina:f314c87.html" title="Last updated on Dec 6, 2022, 8:48 AM UTC (f314c87)">Diff</a>

### CODE REVIEW

**Comment by LeaVerou** [comment]
Hi there, I noticed that the link to the new guide is TODO. I think we should not merge this until that is available, otherwise we're just removing useful guidance.

**Comment by rakina** [comment]
Yep, the plan is to land the BFCache Guide doc first before landing the update to these existing docs.

Is there a good way to get the BFCache Guide doc reviewed? Currently it lives in https://github.com/rakina/bfcache-guide and I can't really make a PR for it because it's a new doc... (How did the Promises guide get reviewed?)

**Comment by rakina** [comment]
Friendly ping for the review for this and the [other PRs](https://github.com/w3ctag/design-reviews/issues/628#issuecomment-1227165134). Also still don't know where to put the new BFCache doc for review... (@domenic, how did you get to make the https://github.com/w3ctag/promises-guide repo?)

**Comment by domenic** [comment]
> (@domenic, how did you get to make the https://github.com/w3ctag/promises-guide repo?)

I think I created a separate repo under my own GitHub, and eventually transferred it to the w3ctag. So you're on the right track...

File: index.bs
Comment (cynthia) [inline index.bs:705]:
```suggestion
[Supporting BFCached Documents](https://w3ctag.github.io/bfcache-guide/) guide.  
```

File: index.bs
Comment (cynthia) [inline index.bs:699]:
```suggestion
* Interacts with a document from the "outside"
  (e.g. sends information to a document)
```

File: index.bs
Comment (cynthia) [inline index.bs:704]:
```suggestion
Specify how your feature works
with non-[=Document/fully active=] BFCached documents,
following the guidelines in the
```

File: index.bs
Comment (cynthia) [inline index.bs:708]:
```suggestion
such as when the iframe
holding the document gets detached.
```

File: index.bs
Comment (cynthia) [inline index.bs:709]:
```suggestion
This guidance only focuses
on the case where BFCache is involved,
```

File: index.bs
Comment (cynthia) [inline index.bs:710]:
```suggestion
and not other cases that might
cause a document to become non-[=Document/fully active=].
```

File: index.bs
Comment (cynthia) [reply index.bs:699]: Maybe "Interacts with an external document" could be better? Quotes tend to implicitly suggest that the quoted term is pseudo-something.

**Comment by cynthia** [comment]
Sorry this took so long, overall the proposed changes look fine to us - and we'd like to do a review of the BFCache guide seprately.

As for that, the repository is here: https://github.com/w3ctag/bfcache-guide
@rakina and @domenic should have write access (on top of the usual TAG suspects)

Wondering how we should review the bfcache-guide repository though, I suppose I could create a PR against the main branch so that we have something to review. Thoughts?

File: index.bs
Comment (torgo) [inline index.bs:700]:
```suggestion
that falls into the
```

**Review by torgo — APPROVED** [review]
looks good to me.

**Review by cynthia — APPROVED** [review]

**Comment by rakina** [comment]
Thanks @cynthia and @torgo! Looks like https://github.com/w3ctag/bfcache-guide already has the doc committed so maybe we should move that into a separate branch and clear out the main branch, and then make a PR against the main branch?

**Pull request merged**
